etstool.py

```diff
@@ -163,8 +164,15 @@ def install(edm, runtime, toolkit, environment, source):
         "{edm} install -y -e {environment} " + packages,
         "{edm} run -e {environment} -- pip install -r ci-src-requirements.txt --no-dependencies",
         "{edm} run -e {environment} -- python setup.py clean --all",
-        "{edm} run -e {environment} -- python setup.py install",
     ]
+    if quiet:
+        commands.append(
+            "{edm} run -e {environment} -- python setup.py install --quiet"
+        )
+    else:
+        commands.append(
+            "{edm} run -e {environment} -- python setup.py install"
+        )
```

Comment:
Oops, just saw this. I think this now needs to resolve merge conflicts with #622

Reply:
#622 didn't remove the original `python setup.py install` (it was doing install twice) which is why i don't think there were merge conflicts.

@aaronayres35 it'll probably be better to integrate with the approach that kit introduced in #622. Like the `if editable:` conditional, we'd just add `--quiet` to `install_pyface` below if `quiet` is set.

Does that make sense?